gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -2501,22 +2508,31 @@ def delete_report(self, report_id: str) -> Any:
         return self._send_xml_command(cmd)
 
     def delete_report_format(
-        self, report_format_id: str, *, ultimate: Optional[bool] = False
+        self,
+        report_format_id: Optional[Union[str, ReportFormatType]] = None,
+        *,
+        ultimate: Optional[bool] = False
     ) -> Any:
         """Deletes an existing report format
 
         Arguments:
             report_format_id: UUID of the report format to be deleted.
+                              or ReportFormatType (enum)
             ultimate: Whether to remove entirely, or to the trashcan.
         """
         if not report_format_id:
             raise RequiredArgument(
-                function=self.delete_report_format.__name__,
+                function=self.verify_report_format.__name__,
```

Comment:
I think this change is wrong. It should still be 
```suggestion
                function=self.delete_report_format.__name__,
```